.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: "go"
+
+go:
+  - "1.9.x"
+  - "1.10.x"
+
+os:
+  - "linux"
+  - "osx"
+
+go_import_path: crawshaw.io/sqlite
+
+install:
+  - go get -t -v crawshaw.io/iox
```

Comment:
This works around https://github.com/crawshaw/sqlite/issues/22#issue-337218094

Reply:
You should be able to drop this now.